.dockerignore

```diff
@@ -0,0 +1,9 @@
+.dockerignore
+.env
+.git
+.gitignore
+.vs
+.vscode
+*/bin
+*/obj
+**/.toolstarget
```

Comment:
@FilipTeply why do you have a `.dockerignore` file?

Reply:
I removed .dockerignore file.